example-mojaloop-backend/values.yaml

```diff
@@ -13,26 +20,6 @@ kafka:
   ##
   affinity: {}
 
-  ## @param nodeSelector Node labels for pod assignment
```

Comment:
Why was this removed?

Reply:
I tried to make the values files more accessible and easier to use by defaulting to the bitnami config. 
just checking and yep it is already set in the base bitnami kafka chart